loops.js

```diff
@@ -51,3 +64,8 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++)
+    if (i>5) {console.log(">> " + i);}
+    else if (i>2) {console.log("> "+i);}
+    else {console.log(i+" ")}
+
```

Comment:
Hallo OLga 
Ik kan eigenlijk geen fouten of errors vinden in je code dus hebt de opdracht goed gemaakt volgens mij.
Groeten Thijs